### PR TITLE
Fix/utils

### PR DESCRIPTION
**Pull request by ThijsZijdel**
Check out the comments in the wdym commit. 
Would u like to have those implemented like i did in the fix/sql history.

Or are we thinking about a actual db log?

Will be working on methods to store api keys, got some ideas for Ai tools.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
-# DEVTOOLS-X
 
-`devutils` is macOS-only, and `devtoys` is Windows-only.  
-So here's DevTools-X -- an x-platform collection of dev-tools that is lighter, safer, and full of feature rich (currenltly 32 modules and growing) development utilities.
+<h1>
+    <img src="assets/logo.min.png" alt="Logo" height="20">
+    <span>DEVTOOLS-X</span>
+</h1>
+
+**DevTools-X** is a cross-platform collection of development utilities, designed to be lightweight, secure, and feature-rich. Unlike `devutils` (macOS-only) and `devtoys` (Windows-only), DevTools-X works seamlessly across multiple operating systems. With over 40 modules and counting, it provides a powerful toolkit for developers everywhere.
```
Comment (ThijsZijdel) [inline README.md:7]: I think that this shifts the focus more directly onto DevTools-X, while still acknowledging the inspiration from other tools like `devutils` and `devtoys`.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 
 > Note: Application is under development, expect some bugs.
 
 > In any case if the app doesn't render/load anything, just delete the file if exists or create settings.json if doesn't exist at: https://docs.rs/tauri/latest/tauri/api/path/fn.data_dir.html
 
-![Logo](assets/banner.png)
```
Comment (ThijsZijdel) [inline README.md:10]: Removed the big logo here, so new users see the app immediately;
Screenshot 2024-08-09 at 17 17 13

Comment (Sparkenstein) [reply README.md:10]: In my opinion, keep it. it's the part of branding. the banner is visible in the meta when someone is searching for it, plus the page looks more professional with a banner.

File: README.md (hunk 1 above)
Comment (Sparkenstein) [reply README.md:7]: This is perfect, feel free to remove the text below it: "Note: Application is under development, expect some bugs." :P it's stable IMO we can remove it

File: README.md (hunk 2 above)
Comment (Sparkenstein) [reply README.md:10]: image

you can move it at the top something like this if you want.

**Comment by ThijsZijdel** [comment]
Oh, I only wanted to close the latest commit

**Pull request closed without merging**